Testing Your Seed.md

```diff
@@ -0,0 +1,34 @@
+# Testing Your Seed
+
+Once you have made the backup of a Seed, this Seed can be used to access your Skywallet. 
+
+**NOTE - The Seed generated for a Skywallet can only be used with a Skywallet. You cannot use this Seed in the Desktop Wallet.**
+
+The **Confirm Seed** button in the Hardware Wallet window of your Desktop Wallet can help you in such situations to verify whether a Seed belongs to a particular Skywallet.
+
+The "Confirm Seed" button will not be available when you are using your Skywallet for the first time. Instead you will be having the button **Create Backup** and once you have clicked on the **Create Backup** button on the Hardware Wallet window and completed the process of the Seed backup, "Create Backup" button will be replaced by the "Confirm Seed" button.
```

Comment:
Please split this sentence up. It is very long. Also replace `having the button` with `see the button`